portals/devportal/src/main/webapp/source/src/app/components/Shared/AppsAndKeys/KeyConfiguration.jsx

```
@@ -407,112 +423,148 @@ const KeyConfiguration = (props) => {
+                                        </TableCell>
+                                    </TableRow>
+
+                                    {/* App Configurations */}
+                                    {applicationConfiguration.length > 0 && applicationConfiguration.map((config) => (
+                                        <AppConfiguration
+                                            config={config}
+                                            previousValue={getPreviousValue(config)}
+                                            isUserOwner={isUserOwner}
+                                            handleChange={handleChange}
+                                            subscriptionScopes={subscriptionScopes}
+                                        />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)